docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,126 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+<box type="tip" seamless>
+The list icons can be customized by adding an {icon="..."} at the end of a list item or a list itself.
+</box>
+
+
+<div id="main-example-markbind">
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="x"}
+* item 2 {icon="heavy_check_mark"}
+* item 3
+* item 4 {icon="/images/deer.jpg"}
+  
+{icon="fas fa-check"}
+</variable>
+</include>
+
+</div>
+
+<box type="warning">
+
+1. Item-level specifications should only be attached to the list heading or list item, not the list content.
+2. A blank line preceding the content is crucial.
+3. It's important to insert a blank line before list-level specifications.
+4. It is possible to any of the icons supported by MarkBind (e.g., Font Awesome, Octicons, Glyphicons, Images and etc.) as the icon of a list item.
+5. Images will maintain a 1 rem bottom margin.
```

Comment:
We should add appropriate padding to all elements, but that is an internal detail.

Reply:
Padding is only added to images to avoid disrupting the original list component's behavior. The original list component uses bullet points, which naturally create spacing. Conversely, image icons can fill an entire square's height and width, potentially leading to display issues.

<img width="182" alt="image" src="https://github.com/MarkBind/markbind/assets/30100720/cebe86ba-3d19-4049-85c4-27fad061e18f">

Furthermore, if the icon size exceeds the line height, it will yield the same issues as mentioned above.

I will refine the design to make it look nicer

Reply:
@SPWwj Yes, we need to tweak the padding to make it look nice when using icons or images as bullet points. Nevertheless, it is not something we need to mention in the UG.